### PR TITLE
BaSP-TG-39-Projects-Axel

### PR DESCRIPTION
**Pull request by FiolAxel**
1. Testing for Post
2. Testing for Put
3. Testing for Delete

### CODE REVIEW

File: src/seeds/projects-seeds.js [hunk 1]
```diff
@@ -1,30 +1,28 @@
 import mongoose from 'mongoose';
 
 export default [{
-  _id: mongoose.Types.ObjectId('6283b662e53ed4c648db8a45'),
+  _id: mongoose.Types.ObjectId('628cf4a2a7b1b8d0ef02360d'),
   name: 'Lionel Messi',
   description: 'this is the description',
-  starDate: 2022,
+  startDate: '2022',
```
Comment (ArianaMaldonado) [inline src/seeds/projects-seeds.js:7]: You need to change all 'star' for 'start' and make sure is not causing you any problems. Every time is different. Start* is the correct form.

File: src/tests/projects.test.js [hunk 2]
```diff
@@ -7,7 +9,49 @@ beforeAll(async () => {
...
+      employees: [
+        {
+          name: '628d6db998a9749e0a37c9bf',
+        },
+      ],
+    });
+    expect(response.body.msg).toEqual('success');
+    projectId2 = response.body.data._id;
+  });
+  test('msg is equal to status success', async () => {
```
Comment (ArianaMaldonado) [inline src/tests/projects.test.js:50]: The test decription does not match with the proper test.

Comment (EzeRocco) [reply src/tests/projects.test.js:50]: the description is wrong, if you don't believe anything the status should be 400

File: src/tests/projects.test.js [hunk 3]
```diff
@@ -31,80 +75,33 @@ describe('GET /projects', () => {
...
+describe('DELETE /projects/:id', () => {
+  test('The project was deleted', async () => {
+    const response = await request(app).delete(`/projects/${projectId}`).send();
+    expect(response.status).toBe(200);
+  });
+  test('Response should return 404 status error', async () => {
+    const response = await request(app).delete('/projects/').send();
+    expect(response.status).toEqual(404);
+  });
+  test('Should return error: false', async () => {
```
Comment (lucasrendo) [inline src/tests/projects.test.js:103]: Why should return error: false?

File: src/tests/projects.test.js [hunk 4]
```diff
@@ -31,80 +75,33 @@ describe('GET /projects', () => {
...
-//     });
-//     expect(response.body).toHaveLength(response.length);
-// });
-// });
+describe('DELETE /projects/:id', () => {
+  test('The project was deleted', async () => {
+    const response = await request(app).delete(`/projects/${projectId}`).send();
+    expect(response.status).toBe(200);
+  });
+  test('Response should return 404 status error', async () => {
```
Comment (lucasrendo) [inline src/tests/projects.test.js:99]: Why should return a 404

File: src/tests/tasks.test.js [hunk 5]
```diff
@@ -1,183 +1 @@
...
-  test('should indicate that the task does not exist - because it was deleted', async () => {
-    const response = await request(app).put(`/tasks/${newTask}`).send(
-      {
-        description: 'Trying to figure test out',
-        workedHours: '10',
-      },
-    );
-    expect(response.body.msg).toEqual('Code 400: This task do not exist');
-  });
-});
```
Comment (lucasrendo) [inline src/tests/tasks.test.js:183]: This should not be deleted

**Review by lucasrendo — CHANGES_REQUESTED** [review]
The tests from tasks were deleted and tests description weren't descriptive

File: src/tests/tasks.test.js (hunk 5 above)
Comment (FiolAxel) [reply src/tests/tasks.test.js:183]: ESO ES DE TASK

**Review by lucasrendo — APPROVED** [review]

**Review by lucianofrailelopez — APPROVED** [review]

**Pull request merged**